dist/common/scripts/scylla_util.py

```diff
@@ -58,6 +58,9 @@ def is_nonroot():
 def is_offline():
     return Path(scylladir_p() / 'SCYLLA-OFFLINE-FILE').exists()
 
+def is_machine_image():
+    return Path(scylladir_p() / 'scylla-machine-image').exists()
```

Comment:
I think it's incorrect to have cross-dependencies between scylla-machine-image and scylla. One can depend on the other but it becomes a mess if both depend on each other.

One thing we can do is move _all_ the cloud code into scylla-machine-image. This way scylla-machine-image detects the cloud and the disk and sets up io-properties.conf. scylla_setup then finds that io-properties.conf is already there and skips iotune. We have way less updates that need to be synchronized across repositories.

/cc @slivne